Styling/README.md

```diff
@@ -622,6 +624,31 @@ Converts the number from radians to degrees.
 }
 ```
 
+#### clamp
+
+`clamp(value : Number,  min : Number, max : Number) : Number`
+
+Returns value if it is greater than min and less than max. Otherwise, if value is less than min, min is returned, and if value is greater than max, max is returned.
+
+```json
+{
+    "show" : "clamp(${Angle}, 0.0, 90.0) > 45.0"
```

Comment:
This example doesn't really show off the function because `"show" : "${Angle} > 45.0"` will produce the same result.